auth/service_test.go

```diff
@@ -82,10 +104,10 @@ func userWithPolicies(t *testing.T, s auth.Service, policies []*model.Policy) st
 
 func TestDBAuthService_ListPaged(t *testing.T) {
 	const chars = "abcdefghijklmnopqrstuvwxyz"
-	db, _ := testutil.GetDB(t, databaseURI)
-	defer db.Close()
+	adb, _ := testutil.GetDB(t, databaseURI)
```

Comment:
Why `adb` now?

Reply:
db package name :) (a database) - not creative?

Reply:
ahhhh.... `db`....